.github/workflows/continous-integration.yml

```diff
@@ -139,12 +139,18 @@ jobs:
       with:
         poetry-version: ${{ env.POETRY_VERSION }}
 
+    - name: Set up virtual environment (Windows)
+      if: matrix.os == 'windows-latest'
+      run: |
+        poetry config virtualenvs.in-project true
```

Comment:
Let's add a comment why we are doing this.

Reply:
Did that.